
\documentclass{article}

% Page geometry settings
\usepackage{geometry}
\geometry{letterpaper, portrait, margin=.75in}

\usepackage{enumitem}
\usepackage{bibentry}

% Define constants
\newcommand{\name}{Sedat Demiriz}
\newcommand{\email}{sedatdemiriz97@gmail.com}
\newcommand{\phone}{778-866-9440}
\newcommand{\website}{sdemiriz.github.io}

% Remove page numbers
\thispagestyle{empty}

% Add custom commands for reused document sections
\newcommand{\experienceheader}[3]{\item \textbf{#1} - \textit{#2} \hfill \textit{#3} \vspace{-.2em}}
\newcommand{\customsection}[1]{\section*{#1} \vspace{-1em} \hrulefill \vspace{-.5em}}

\begin{document}

% Set up the header
\begin{minipage}[t]{1\textwidth} 
  \centering \Huge 
  \textbf{\name} \vspace{.5em} 
\end{minipage}
\begin{minipage}[t]{1\textwidth} 
  \centering \normalsize 
  \phone\hspace{0.5em}-\hspace{0.5em}\email\hspace{0.5em}-\hspace{0.5em}\website
\end{minipage}

% Professional Summary
\section*{Professional Summary} \vspace{-1em} \hrulefill \vspace{.5em}

\noindent Bioinformatics Programmer with 3 years of experience. Specialty areas include application of reproducible
software development methods to solving research problems, identifying associations in genotyping and NGS data from 
general and disease population cohorts

% Experience
\setlist[itemize, 1]{label=\!, leftmargin=0em}
\setlist[itemize, 2]{label=\textbf{-}, leftmargin=1.2em, itemsep=.05em}

\customsection{Experience}
\begin{itemize}
  \experienceheader{Computational Biologist}{Sequence Bio Inc.}{Jul '21 -- Nov '23}
    \begin{itemize}
      \item Lead the development of scalable and reproducible GWAS and ROH analysis pipelines to identify genotype-phenotype associations in company's Newfoundland founder population cohort
      \item Implemented standalone and pipeline step scripts for genomic data analysis and visualization to identify genes of interest for an international collaboration with pharma company
      \item Combined internal and public data sources to characterize the distribution of the founder effect across Newfoundland and its relation to source populations in Ireland and Britain
    \end{itemize}
  \experienceheader{Research Programmer}{BC Genome Sciences Center}{Nov '20 -- Mar '21}
    \begin{itemize}
      \item Performed QC on inbound AML NGS data using standard NGS quality metrics to ensure clean data input for lab's internal analysis platform
      \item Streamlined existing QC and visualization scripts and refactored for flexibility of use for future QC efforts
      \item Addressed graduate student questions about ways of solving AML research questions by showcasing scripting and command line approaches to tackling data transformations 
    \end{itemize}
  \experienceheader{Biosensor R\&D Co-op Programmer}{CiBER Lab, SFU}{May -- Sep '18}
    \begin{itemize}
      \item Automated biosensor voltage response curve data processing and visualization for display and diagnosis of individual sensor instances to identify issues in the fabrication process
      \item Screened hundreds of sensor instances during development to achieve sample sizes necessary for confidence in manufacturing 
      \item Implemented and presented metrics for determining biosensor instance success based on response pattern to inform lab heads on progress made on the development of the manufacturing process
    \end{itemize}
  \experienceheader{Genetics Co-op Student}{Istanbul University Genetics Institute}{May -- Aug '16}
    \begin{itemize}
      \item Carried out patient PCR, DNA and RNA isolation and sample QC procedures for input clinical NGS diagnostics
      \item Worked routinely in cell culture lab to grow five varieties of tumor cell populations for cancer studies at the lab
      \item Lead and supervised four other co-op students during their first month with the lab to ensure good integration
    \end{itemize}
\end{itemize}

% Skills
\setlist[itemize, 1]{label=\textbf{-}, leftmargin=1.2em, itemsep=.05em}

\customsection{Skills}
\begin{itemize}
  \item Software development experience in Python, R, and Bash using good software development practices: version control, unit testing, documentation 
  \item Writing robust data transformation and analysis scripts and producing easy-to-understand charts and plots to visualize underlying patterns in genomics data
  \item Development of scalable workflows on Linux, leveraging HPC and Cloud resources for processing large NGS datasets using industry standard bioinformatics tools
  \item Identifying phenotype-genotype associations in clinical data from general population cohorts
  \item Querying SQL relational databases and unstructured public data sources to derive insights
\end{itemize}

% Education
\setlist[itemize, 1]{label=\!, leftmargin=0em}
\setlist[itemize, 2]{label=\textbf{-}, leftmargin=1.2em, itemsep=0.05em}

\customsection{Education}
\begin{itemize}
\experienceheader{BSc Joint Major Mol. Biology and Biochem. \& Comp. Sci.}{Simon Fraser University}{'15 -- '20}
  \begin{itemize}
    \item Computing Science: Data Science, Machine Learning, Databases, Algorithms
    \item Molecular Biology: Bioinformatics, Human Genomics, Applied Wet Labs
    \item Statistics: Exploratory Data Analysis, Experimental Design and Analysis
  \end{itemize}
\end{itemize}

% Publications
\setlist[itemize, 1]{label=\!, leftmargin=0em}

\bibliographystyle{apalike}
\nobibliography{publications}
\customsection{Publications}
\begin{itemize}
  \item \textbf{-} \bibentry{nl_pop_descends_2023}
  \item \textbf{-} \bibentry{nl_y_chrom_2022}
\end{itemize}

\end{document}
